apps/indexer/src/handlers/AlloV1ToV2ProfileMigration.ts

```
@@ -0,0 +1,6 @@
+// /*
+//  * Please refer to https://docs.envio.dev for a thorough guide on all Envio indexer features
+//  */
+import { AlloV1ToV2ProfileMigration } from "../../generated";
+
+AlloV1ToV2ProfileMigration.ProfileMigrated.handler(async ({}) => {});
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)